implementations/contracts/ERC725/ERC725Account.sol

```diff
@@ -21,108 +12,32 @@ import "../ILSP1/ILSP1_UniversalReceiverDelegate.sol";
  *  @author Fabian Vogelsteller <fabian@lukso.network>
  */
 
-// TODO add ERC777, ERC223, ERC721 functions?
-
-contract ERC725Account is ERC725, IERC1271 , ILSP1 {
-    using ERC725Utils for ERC725Y;
-
-    bytes4 internal constant _INTERFACE_ID_ERC1271 = 0x1626ba7e;
-    bytes4 internal constant _ERC1271FAILVALUE = 0xffffffff;
-    bytes4 constant _INTERFACE_ID_LSP1 = 0x6bb56a14;
-    bytes4 constant _INTERFACE_ID_LSP1DELEGATE = 0xc2d7bcc1;
-
-    bytes32 constant private _UNIVERSAL_RECEIVER_DELEGATE_KEY =
-    0x0cfc51aec37c55a4d0b1a65c6255c4bf2fbdf6277f3cc0730c45b828b6db8b47; // keccak256("LSP1UniversalReceiverDelegate")
-
-    event ValueReceived(address indexed sender, uint256 indexed value);
+contract ERC725Account is ERC725, ERC725AccountCore  {
 
     /**
      * @notice Sets the owner of the contract
      * @param _newOwner the owner of the contract.
      */
-    constructor(address _newOwner) ERC725(_newOwner) {
-        // set SupportedStandards > ERC725Account
-        bytes32 key = bytes32(0xeafec4d89fa9619884b6b89135626455000000000000000000000000afdeb5d6); // SupportedStandards > ERC725Account
-        store[key] = abi.encodePacked(bytes4(0xafdeb5d6)); // bytes4(keccak256('ERC725Account')
-        emit DataChanged(key, store[key]);
-
+    constructor(address _newOwner) ERC725(_newOwner){
+        
+        _registerInterface(_INTERFACE_ID_ERC725Account);
         _registerInterface(_INTERFACE_ID_ERC1271);
         _registerInterface(_INTERFACE_ID_LSP1);
     }
 
-    receive() external payable {
-        emit ValueReceived(_msgSender(), msg.value);
+        function execute(
```

Comment:
please remove the intendation.
Why adding a new function here?

Reply:
Derived contract must override function "execute". Two or more base classes define function with same name and parameter types.